_posts/2018-02-15-upgrading-a-monolith.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Upgrading a monolith from Rails 4.0 to 5.1"
+date: 2018-02-15
+categories: ["rails", "upgrades", "case-study"]
+author: "emily"
+---
+
+We recently completed work with [Power Home Remodeling](https://powerhrg.com) on a Rails upgrade for their self-described “monolith CRM/ERP application”, and were able to speak to them about their experience with Ombu Labs.
+
+<!--more-->
+
+We talked to [Ben Langfeld](https://www.linkedin.com/in/benlangfeld/), the Application Solution Architect at [Power Home Remodeling](https://powerhrg.com), about the work on their app, Nitro. According to Ben, their monolithic CRM/ERP application is continuously built by a team of approximately 50 developers, system administrators, testers and support staff. The application contains *over 500,000 lines of Ruby on Rails and Javascript code*, and by their own definition, is one of the more complex Rails applications out there in terms of scope.
```

Comment:
@ercohen14 It sounds better like this `Ben Langfeld, Application Solutions Architect at...`

Maybe `most complex Rails applications out there`?